Chapter2_ProgramStructure/Chapter 2 #2.js

```diff
@@ -0,0 +1,23 @@
+function f3()
+{
+    var i;
+    for(i=1;i<=100;i++)
+    {
+        if(i%3==0)
+        {
+            if(i%5==0)
+            {
+                console.log("FizzBuzz");
+            }
+            else
+            {
+                 console.log("Fizz");
+            }
+        }
+        else if(i%5==0)
+        {
+            console.log("Buzz");
+        }
+    }
```

Comment:
You will need an else block. What happens to numbers that are not divisible by either 3 or 5?